fl4health/clients/basic_client.py

```diff
@@ -1185,3 +1193,14 @@ def maybe_progress_bar(self, iterable: Iterable) -> Iterable:
                 "bar_format": f"{LOG_COLORS['INFO']}INFO{LOG_COLORS['RESET']}" + " :        {l_bar}{bar}{r_bar}",
             }
             return tqdm(iterable, **kwargs)
+
+    def transform_gradients(self, losses: TrainingLosses) -> None:
```

Comment:
Great addition! This can be leveraged by the scaffold client to simplify the code. I put a [ticket](https://app.clickup.com/t/8689drd6a) in the backlog for me to port the client to leverage this once this PR is merged in.